src/ParquetMsg.chpl

```diff
@@ -1000,7 +994,12 @@ module ParquetMsg {
 
     var pqErr = new parquetErrorMsg();
 
-    if c_writeListColumnToParquet(filename.localize().c_str(), c_ptrTo(locOffsets), c_ptrTo(localVals),
+    var valPtr: c_void_ptr = nil;
+    if localVals.size != 0 {
+      valPtr = c_ptrTo(localVals);
+    }
+
+    if c_writeListColumnToParquet(filename.localize().c_str(), c_ptrTo(locOffsets), valPtr,
```

Comment:
is there a scenario where we might wanna do something similar with `c_ptrTo(locOffsets)`?

Reply:
@pierce314159 - No there should not be. In those cases the intersection size will be 0 and the write will not run.